Light Vox Engine/Graphics/GraphicsCore.cpp

```diff
@@ -373,6 +383,55 @@ HRESULT GraphicsCore::InitGeometryPSO()
     return S_OK;
 }
 
+#ifdef _DEBUG
```

Comment:
Does this mean that we can only draw the wireframes in debug mode? Maybe we can add in another #define for drawing the wireframes ?

Reply:
it's only debug because it's only used for debugging